src/commands/optimize.ts

```diff
@@ -191,4 +199,23 @@ export default class Optimize extends Command {
     }]);
     this.outputMethod = outputRes.output;
   }
+
+  private async collectMetricsData(report: Report) {
```

Comment:
We could simplify this function and get just all properties from the report instead of manually checking.

```ts 
for (const reportOptimization in report) {
      if (reportOptimization.length) {
        this.metricsMetadata[`optimization_${reportOptimization}`] = true;
      }
    }

```

Not sure if will work

Reply:
It works! 👍